recipe/install.sh

```diff
@@ -0,0 +1,9 @@
+make install PREFIX=${PREFIX}
+
+if [[ "$PKG_NAME" == *static ]]
+then
+	# relying on conda to dedup package
+	echo "Keeping all files, conda will dedupe"
```

Comment:
Did you check this? 😂

Reply:
yep, I did! this works, because zstd (non-static) is a host requirement of zstd-static, so the header files and dynamic lib are already there.